study-builder/studies/atcp/snippets/self-consent/consent-contact.conf

```diff
@@ -0,0 +1,50 @@
+{
+  "titleTemplate": {
+    "templateType": "HTML",
+    "templateText": "$at_consent_contact_title",
+    "variables": [
+      {
+        "name": "at_consent_contact_title",
+        "translations": [
+          {
+            "language": "en",
+            "text": "What if I have questions?"
+          }
+        ]
+      }
+    ]
+  },
+  "bodyTemplate": {
+    "templateType": "HTML",
+    "templateText": "$at_consent_contact_detail",
+    "variables": [
+      {
+        "name": "at_consent_contact_detail",
+        "translations": [
+          {
+            "language": "en",
+            "text": """
+              If you have any questions, concerns, or complaints please send an email to <a href="mailto:support@atfamilies.org"
```

Comment:
At which point should we start thinking about internationalization? Although native support for i18n is very lacking in the system right now, we can at the very least start to centralize user-facing strings so they will be easier to translates. I'll suggest consolidating strings into a "substitutions" file, which the Prion study is starting to do.